### PR TITLE
Record deletion

### PR DESCRIPTION
**Pull request by rerowep**
delete record during harvesting

### CODE REVIEW

File: scripts/server [hunk 1]
```diff
@@ -6,9 +6,19 @@
 # RERO Ebooks is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-set -e
+sset -e
```
Comment (jma) [inline scripts/server]: error?

File: scripts/server [hunk 2]
```diff
@@ -6,9 +6,19 @@
...
 
 script_path=$(dirname "$0")
-FLASK_ENV=development invenio run \
-    --cert "$script_path"/../docker/nginx/test.crt \
-    --key "$script_path"/../docker/nginx/test.key
+
```
Comment (jma) [inline scripts/server]: can be a separate commit

Comment (rerowep) [reply scripts/server]: no time
